src/languages/cpp.ts

```diff
@@ -1,109 +1,112 @@
-import {
```

Comment:
Not sure what happened with this diff. Did you change line endings or something? Same with all the other language parse tree pattern files files

Reply:
I'm trying to switch to LF and something rent wrong.